docs/Waypoint.md

```diff
@@ -0,0 +1,87 @@
+# Waypoint Service
```

Comment:
Let's rename this module back to Webhooks.md

it'll make review easier to see diff, but mainly because I believe the concept of Webhooks is what needs to be explained first and foremost, and then Waypoint should be a subheading of Webhooks. We should also introduce the fact that we recommend using NATS first and foremost.